<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="formLayout">
      <item>
       <widget class="QLineEdit" name="titleEdit">
        <property name="placeholderText">
         <string>Judul</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="categoryEdit">
        <property name="placeholderText">
         <string>Kategori</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="statusCombo">
        <item>
         <property name="text">
          <string>Pending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Selesai</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Tambah</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#5dade2; color:white; padding:5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="taskTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_CSV"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
  </widget>
  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
